src/Service/S3/src/ValueObject/EventBridgeConfiguration.php

```diff
@@ -0,0 +1,25 @@
+<?php
+
+namespace AsyncAws\S3\ValueObject;
+
+/**
+ * Enables delivery of events to Amazon EventBridge.
+ */
+final class EventBridgeConfiguration
+{
+    public function __construct(array $input)
+    {
+    }
+
+    public static function create($input): self
+    {
+        return $input instanceof self ? $input : new self($input);
+    }
+
+    /**
+     * @internal
+     */
+    public function requestBody(\DOMElement $node, \DOMDocument $document): void
+    {
+    }
```

Comment:
looks like AWS definition is empty:

https://github.com/aws/aws-sdk-php/blob/ade6621f709261097a0fd2ca0cad7a15ba899b76/src/data/s3/2006-03-01/api-2.json#L2642-L2646

I'm not sure what is the expected output for an empty tag :thinking: 
Let's wait few days if AWS adds members in the shape.  Otherwise, I'll update the generators to handle this case.